docs/index.rst

```diff
@@ -26,11 +26,18 @@ Girder: a data management platform
 What is Girder?
 ---------------
 
-Girder is a free and open source web-based **data management platform** developed by
+Girder is a libre open source web-based **data management platform** developed by
```

Comment:
"free" is ambiguous: free as in beer vs. free as in freedom. Libre was the word created to express the second form precisely. I don't have strong feelings about this wording, I'm open to suggestions. If we keep libre, I'm certainly OK with a link to its meaning.